Add warning in front of generated code blocks

szarnyasg opened the pull request.

Mause commented on scripts/README.md:
````diff
@@ -0,0 +1,26 @@
+build/debug/duckdb -c "INSTALL 'build/debug/extension/httpfs/httpfs.duckdb_extension'; INSTALL 'build/debug/extension/icu/icu.duckdb_extension';"
+```
+
+Install the NodeJS dependencies:
````
Might be worth documenting the python dependencies too

szarnyasg commented:
@Mause I also documented the DuckDB (extensions/CLI/Python) instructions in [e2ef64e](https://github.com/duckdb/duckdb-web/pull/1223/commits/e2ef64e8721421d07d61430a299aafd15e57acc3), can you please take a look?

Mause commented on scripts/README.md:
````diff
@@ -0,0 +1,34 @@
+
+```bash
+# build the DuckDB CLI client
+GEN=ninja BUILD_HTTPFS=1 BUILD_ICU=1 OPENSSL_ROOT_DIR=/opt/homebrew/bin/ make debug
````
What's the rationale behind directing people to build from scratch instead of installing dev builds from pypi?

szarnyasg replied:
To be frank, it didn't occur to me to use dev builds in the first place. That's indeed much simpler and definitely the recommended approach.

szarnyasg replied:
Pushed a commit that recommends the nightly build.

Mause commented on scripts/README.md:
````diff
@@ -0,0 +1,34 @@
+Run the script as follows:
+
+```bash
+scripts/generate_all_docs.sh <path_to_duckdb_dir>
````
```suggestion
./scripts/generate_all_docs.sh <path_to_duckdb_dir>
```

Mause commented on scripts/README.md:
````diff
@@ -0,0 +1,34 @@
+
+```bash
+npm install
+python -m pip install -r requirements.txt
````
```suggestion
pip install -r requirements.txt
```

Mause commented on scripts/README.md:
```diff
@@ -0,0 +1,34 @@
+
+# build the DuckDB Python client
+python3 tools/pythonpkg/setup.py install
+# extensions are shared across clients so there is no need to install them to Python
```
This isn't always true on Linux due to abi issues

Mause commented on scripts/README.md:
```diff
@@ -0,0 +1,34 @@
+build/debug/duckdb -c "INSTALL 'build/debug/extension/icu/icu.duckdb_extension';"
+
+# build the DuckDB Python client
+python3 tools/pythonpkg/setup.py install
```
```suggestion
pip install -e tools/pythonpkg
```

szarnyasg commented:
Thanks for the review, I addressed its points. The question is, do we even keep the manual installation instructions for the package? I have removed them for now.
